pages/services/include/build-local-universe-ee-only.tmpl

```diff
@@ -0,0 +1,311 @@
+
+    ```bash
+    $ sudo make DCOS_VERSION=<your DC/OS version> DCOS_PACKAGE_INCLUDE="cassandra:1.0.25-3.0.10,cassandra:1.0.24-3.0.10,{{ model.packageName }}:1.4.2" local-universe
+    ```
+
+<a name="building-your-own-staticbase"></a>
+### Building Your Own, off a non-changing universe-static base image.
+
+Mesosphere provides a `mesosphere/universe-static` Docker image, which has all of the core requirements to run a local universe.  All that must be added are your own certificates and rep contents.
+
+1. Use the `make` command to download the static image:
+
+    ```bash
+    $ sudo make static-online
+    ## Will pull and re-tag mesosphere/universe-static to universe-static
+    ```
+
+1. Create your certs (either use `make certs` or generate your own), and add them to the static image to create a `universe-base` image:
+
+    ```bash
+    $ sudo make static-base
+    ## Will add certs to universe-static and create universe-base
+    ```
+
+1. Add content (see above):
+
+    ```bash
+    $ sudo make DCOS_VERSION=<your DC/OS version> local-universe
+    ```
+
+To generate your own static image, use and/or modify the make target `static-build` (e.g., `make static-build` instead of `make static-online`)
+
+This leaves three options:
+
+```bash
+make base
+make DCOS_VERSION=<your DC/OS version> local-universe
+```
+
+```bash
+make static-online
+make static-base
+make DCOS_VERSION=<your DC/OS version> local-universe
+```
+
+```bash
+make static-build
+make static-base
+make DCOS_VERSION=<your DC/OS version> local-universe
+```
+
+<a name="building-old-version"></a>
+### Building an old version of Local Universe
+The latest version of Local Universe has changed directory structure in a way that is not compatible with the old version of Local Universe. If you are interested in creating an old version of Local Universe with the old directory structure you must use the following command instead of the `local-universe` make target describe previously.
+
+```bash
+$ sudo make DCOS_VERSION=<your DC/OS version> old-local-universe
+```
+
+<a name="outside-resources"></a>
+### Outside Resources
+
+As a workaround for the image and CLI resource issues in [the FAQ](#faq), you can place those assets outside of the cluster.
+
+1. Place your CLI and image resources on a web server accessible to CLI and web UI users.
+
+2. Edit the URLs in the `resource.json` file of your package of interest to point to each of those resources, in the `images` and `cli` sections.
+
+3. Edit Makefile so the call to `local-universe.py` includes arguments `--nonlocal_images` and `--nonlocal_cli`.
+
+4. Proceed with [Building Your Own](#building-your-own) as above.
+
+<a name="running-local"></a>
+## Running Local Universe as a Marathon Service
```

Comment:
Now I am at loss - are we describing here the way to add Edgelb and other enterprise-only packages to a local universe or are we describing how to build local universe? Maybe we should split the part about building local universe into a separate document and concentrate here only on the issue described in COPS-3321?

This comment also applies to two previous sections about building the universe package.